recipes/psiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "psiflow" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molmod/psiflow/archive/v{{ version }}.tar.gz
+  sha256: c6103905ecfe9ded67a1c52142424f184e50c24598998be931f25d5a13b6ae43
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
```

Comment:
It seems unusual to have an entry point that comes in through a dependency. What's the reason for doing this?

Reply:
Sorry this was a mistake from my copy & pasting.